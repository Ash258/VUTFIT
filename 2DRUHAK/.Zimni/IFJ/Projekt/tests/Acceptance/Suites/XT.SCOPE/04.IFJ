' Declaration in all condition branches

scope
	dim i as integer = 24 * 2

	do while i > 0
		if i > 38 then
			dim b as double = 42
			print b;
		else
			if i > 24 then
				dim b as double = 74
				print b;
			else
				if i > 10 then
					dim b as double = 13
					print b;
				else
					dim b as double = 11
					print b;
				end if
			end if
		end if
		print !"\n";
		i = i - 1
	loop
end scope
