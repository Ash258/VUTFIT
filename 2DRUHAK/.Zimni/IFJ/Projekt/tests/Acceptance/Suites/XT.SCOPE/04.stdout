 42
 42
 42
 42
 42
 42
 42
 42
 42
 42
 74
 74
 74
 74
 74
 74
 74
 74
 74
 74
 74
 74
 74
 74
 13
 13
 13
 13
 13
 13
 13
 13
 13
 13
 13
 13
 13
 13
 11
 11
 11
 11
 11
 11
 11
 11
 11
 11
